package.json

```diff
@@ -1,56 +1,60 @@
 {
-  "name": "git-task-list",
-  "version": "0.0.0",
-  "private": true,
-  "description": "Small description for gsoc-prep-tasks goes here",
-  "license": "GPL-3.0-or-later",
-  "author": "",
-  "directories": {
-    "doc": "doc",
-    "test": "tests"
-  },
-  "repository": "",
-  "scripts": {
-    "build": "ember build",
-    "lint:js": "eslint ./*.js app blueprints config lib server tests",
-    "start": "ember serve",
-    "test": "ember test"
-  },
-  "devDependencies": {
-    "active-model-adapter": "^2.1.1",
-    "broccoli-asset-rev": "^2.4.5",
-    "ember-ajax": "^3.0.0",
-    "ember-browserify": "^1.1.13",
-    "ember-cli": "~3.1.4",
-    "ember-cli-app-version": "^3.2.0",
-    "ember-cli-babel": "^6.6.0",
-    "ember-cli-dependency-checker": "^2.0.0",
-    "ember-cli-eslint": "^4.2.1",
-    "ember-cli-htmlbars": "^2.0.1",
-    "ember-cli-htmlbars-inline-precompile": "^1.0.0",
-    "ember-cli-inject-live-reload": "^1.4.1",
-    "ember-cli-qunit": "^4.1.1",
-    "ember-cli-sass": "^7.1.7",
-    "ember-cli-shims": "^1.2.0",
-    "ember-cli-sri": "^2.1.0",
-    "ember-cli-uglify": "^2.0.0",
-    "ember-cli-update": "^0.21.4",
-    "ember-data": "~3.1.0",
-    "ember-maybe-import-regenerator": "^0.1.6",
-    "ember-paper": "^1.0.0-alpha.19",
-    "ember-source": "~3.1.0",
-    "eslint-plugin-ember": "^5.0.0",
-    "github-api": "https://github.com/github-tools/github/archive/v3.1.0.tar.gz",
-    "loader.js": "^4.2.3"
-  },
-  "engines": {
-    "node": "^4.5 || 6.* || >= 7.*"
-  },
-  "dependencies": {
-    "ember-cli-showdown": "^4.4.2",
-    "ember-export-application-global": "^2.0.0",
-    "ember-load-initializers": "^1.1.0",
-    "ember-resolver": "^4.5.5",
-    "minimatch": "^3.0.3"
-  }
+    "name": "git-task-list",
```

Comment:
configure the bear to use 2 space indent, as was asked at https://github.com/coala/git-task-list/pull/42#discussion_r193959988

The bear defaults are not important.  The repositories current style are more important than the bear defaults.  We want minimal change.  Also, your mentors requests are important - try to do them whenever possible, unless you have a very good reason why it can not be done.

Reply:
I thought he wasn't ordering me to use 2 space. I thought he was asking me why. Sorry about that.

Reply:
It wasnt an order.  It was giving you direction.  Now sometimes the direction given is wrong, and it is ok to reply with a technical reason why it is wrong.
But an 'i assume something else is more important' isnt a technical response.